disruption_generator/parsers/experiment_parser.py

```diff
@@ -98,4 +99,6 @@ def _init_actions(element_trigger):
 
             disruption = Disruption(name=name, listener=listener, actions=actions)
 
-            yield disruption
+            _disruptions.append(disruption)
+
+        return _disruptions
```

Comment:
Agree on refactoring to scenario. No need for a Scenario class at the moment since there is no additional metadata related to it. We can consider it in the future if necessary.